eisbuk-install.sh

```diff
@@ -0,0 +1,19 @@
+# install and build shared files package
+cd eisbuk-shared
```

Comment:
Nitpicking: I think this file could benefit from a `set -ex` so that it shows what it does at each step and it fails with an error if any intermediate step fails. Currently the script will exit with the status of the last command.

Reply:
Also, let's set the executable bit and add a shebang to use `/bin/sh`.

Reply:
Yeah, I was thinking about that as well, just didn't know how exactly to implement it. Thanks ;)